countlyCommon/countlyCommon/CountlyBase.cs

```diff
@@ -1024,8 +1036,11 @@ public async Task ChangeDeviceId(String newDeviceId, bool serverSideMerge = fals
         internal bool consentRequired = false;
         internal Dictionary<ConsentFeatures, bool> givenConsent = new Dictionary<ConsentFeatures, bool>();
 
-        public enum ConsentFeatures { Sessions, Events, Location, Crashes, Users, Views };
-
+        public enum ConsentFeatures
+        {
+            Sessions, Events, Location, Crashes, Users, Views, Push, Feedback,
+            StarRating, RemoteConfig
```

Comment:
this should all be on a single line

Reply:
I performed formating, it is also acceptable.